vumi/transports/apposit/tests/test_apposit.py

```diff
@@ -48,11 +49,6 @@ def setUp(self):
         self.transport_url = self.transport.get_transport_url()
         self.web_path = config['web_path']
 
-    @inlineCallbacks
-    def tearDown(self):
-        yield self.mock_server.stop()
-        yield super(TestAppositTransport, self).tearDown()
-
```

Comment:
There are quite a few cleanups like this. Are they necessary or is it just code cleanup?